cherrypy/_cprequest.py

```diff
@@ -929,6 +939,7 @@ def finalize(self):
             # and 304 (not modified) responses MUST NOT
             # include a message-body."
             dict.pop(headers, 'Content-Length', None)
+            self._flush_body()
```

Comment:
@jaraco do you fully agree with such approach? I'd prefer not putting the body there over consuming it here.